### DISCUSSION TITLE
Generate BTF information for kernel version 4.9

### DISCUSSION THREAD

**Discussion by tanshihaj** [aquasecurity/btfhub#87] [Q&A] [answered]
I have a custom linux kernel 4.9 (the one from jetpack) and I want to generate BTF information for it. Is there anybody who understand how to do it?

I've tried to do it with `pahole -J vmlinux` but get following error:
```
$ pahole -J vmlinux
Cannot open vmlinux
Failed to encode BTF
```

I think that I don't understand something.

**Comment by tanshihaj** [comment, marked as answer]
It seems that it was some issue with dwarves version `1.21-0ubuntu1~20.04`. The latest dwarves from upstream (git commit 31bc0d7410572f6e03e3ed9da7c8c6f0d8df23c8) works fine.

**Reply by rafaeldtinoco** [reply]
Thank you for answering, and documenting it here!
